metagenomics.py

```diff
@@ -1241,6 +1284,30 @@ def kraken_build(db, library, taxonomy=None, subsetTaxonomy=None,
 
     if clean:
         kraken_tool.execute('kraken-build', db, '', options={'--clean': None})
+
+
+def parser_kraken_build(parser=argparse.ArgumentParser()):
+    parser.add_argument('db', help='Kraken database output directory.')
```

Comment:
this section looks like unintentionally duplicated code... I moved all the `parser_` methods before the main methods in metagenomics.py to keep it consistent with other top-level python scripts (keeps the description and self-documentation close to the method spec and docstring) and also moved the `__commands__.append` line up next to each main method as well (again, keeping in line with other top level python scripts).